Gruntfile.js

```diff
@@ -85,8 +85,51 @@ module.exports = function(grunt) {
             }
         },
         clean: {
+            coverage: ['coverage/', 'coverage.lcov', '.nyc_output/'],
             dist: ['.dist/'],
+            packages: ['Consus-Client-*/'],
             test: ['.test/']
+        },
+        electron: {
+            win32Build: {
```

Comment:
Youre right, I'll change it.  I just named it that since the platform name is 'win32'